java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -210,6 +211,8 @@ private static Object[] stringCommands(BaseTransaction<?> transaction) {
         transaction
                 .set(stringKey1, value1)
                 .get(stringKey1)
+                .getex(stringKey1)
```

Comment:
this will fail for redis version <6.2.0

Reply:
but in this part of the tests, there's no section for < 6.2.0. should i make another section for this command?

Reply:
yes, its probably that there hasnt been tests here that needed a 6.2 check, you can copy what other test groups are doing